commands/view-state.js

```diff
@@ -36,8 +35,21 @@ async function viewState(options) {
     const { accountId, prefix, finality, blockId, utf8 } = options;
     const near = await connect(options);
     const account = await near.account(accountId);
-
-    let state = await account.viewState(prefix, { blockId, finality });
+    if (finality && blockId) {
+        console.error('Only one of --finality and --blockId can be provided');
+        process.exit(1);
+    } else if (!finality && !blockId) {
+        console.error('Must provide either --finality or --blockId');
+        process.exit(1);
+    }
+    // near-api-js takes block_id instead of blockId
+    let block_id = blockId;
+    if (blockId && !isNaN(Number(blockId))) {
```

Comment:
For parsing of inputs data, we should use .coerce() (ref: https://yargs.js.org/docs/#coerce) -- failures to coerce arguments into our desired datatype should throw an error rather than exiting the process, so Yargs can generate user-friendly, consistent output.

Reply:
Nice! I didn't know this